### ISSUE TITLE
Some tests are flaky

### ISSUE THREAD

**Issue by je-ik** [O2-Czech-Republic/proxima-platform#87]
These tests fail from time to time.
```
testWrite[0](cz.o2.proxima.gcloud.storage.BulkGCloudStorageWriterTest)  Time elapsed: 6.134 sec  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 2000 milliseconds
	at cz.o2.proxima.gcloud.storage.BulkGCloudStorageWriterTest.testWrite(BulkGCloudStorageWriterTest.java:137)

testWrite[1](cz.o2.proxima.gcloud.storage.BulkGCloudStorageWriterTest)  Time elapsed: 6.117 sec  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 2000 milliseconds
	at cz.o2.proxima.gcloud.storage.BulkGCloudStorageWriterTest.testWrite(BulkGCloudStorageWriterTest.java:137)
```

**Comment by je-ik** [comment]
Another one (might be related to euphoria-beam 0.9-SNAPSHOT):
```
testViewPersist(cz.o2.proxima.storage.pubsub.PubSubPartitionedViewTest)  Time elapsed: 0.304 sec  <<< FAILURE!
java.lang.AssertionError: expected:<1> but was:<0>
	at cz.o2.proxima.storage.pubsub.PubSubPartitionedViewTest.testViewPersist(PubSubPartitionedViewTest.java:314)
```

**Comment by je-ik** [comment]
```
java.lang.AssertionError: expected:<3> but was:<4>
	at cz.o2.proxima.storage.kafka.LocalKafkaCommitLogDescriptorTest.testBulkObserveWithExceptionAndRetry(LocalKafkaCommitLogDescriptorTest.java:797)
```

**Comment by je-ik** [comment]
The first two issues are solved via datadrivencz/proxima-platform#81

**Comment by je-ik** [comment]
Fixed.
